partner_statement/__manifest__.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2022 ForgeFlow, S.L. (http://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Partner Statement",
+    "version": "16.0.1.1.0",
```

Comment:
```suggestion
    "version": "16.0.1.0.0",
```

Reply:
Thanks @cuongnmtm I've now made that adjustment and squashed the last three commits back into one migration commit